staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -173,3 +173,15 @@ type ServiceAffinityArgs struct {
 	// +listType=atomic
 	AntiAffinityLabelsPreference []string `json:"antiAffinityLabelsPreference,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeBindingArgs holds arguments used to configure the VolumeBinding plugin.
+type VolumeBindingArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// BindTimeoutSeconds is the timeout in seconds in volume binding operation.
```

Comment:
Copy this to the internal type `pkg/scheduler/apis/config`, except for the exact value.

Reply:
done

Reply:
What about `pkg/scheduler/apis/config/types_pluginargs.go`?

Reply:
my bad, updated now :(